library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/MainActivity.kt

```diff
@@ -39,6 +50,15 @@ internal class MainActivity :
     private val applicationName: CharSequence
         get() = applicationInfo.loadLabel(packageManager)
 
+    private val permissionRequest = registerForActivityResult(
+        ActivityResultContracts.RequestPermission()
+    ) { isPermissionGranted: Boolean ->
+        if (!isPermissionGranted) {
+            showToast(applicationContext.getString(R.string.chucker_notifications_permission_not_granted))
+            Logger.error("Notification permission denied. Can`t show transactions info")
+        }
```

Comment:
Might be worth to also log here a success message?

Reply:
I don't think it is worth it.

Reply:
Ok but you log a:

```
Logger.info("Notification permission granted")
```

Just below. Shoudl we remove that as well then?

Reply:
Good catch. Didn't like empty braces initially. Removed.